src/components/BundleForm.tsx

```diff
@@ -58,16 +54,21 @@ const BundleForm = forwardRef<BundleFormMethods, BundleFormProps>(
 
     const onSubmit = (e: FormEvent) => {
       e.preventDefault();
-      const cost = codeInputRef.current?.getBundleCost?.();
       if (!bundle) {
         toast.error("Bundle JSON not provided.", { autoClose: 3000 });
-      } else if (cost && cost > Number(istBalance) / 10 ** 6) {
+        return;
+      }
+      const cost = codeInputRef.current?.getBundleCost?.();
```

Comment:
I'm not sure I understand the motivation for moving stuff around here, but I can say that it behaves as expected.